style: apply /style-guide pass to sandboxes (#2668)
wandb/docs: sandboxes/mltrain-in-sandbox-tutorial.mdx

source venv/bin/activate  # On Windows, use `venv\Scripts\activate`
```  */}

### Install the W&B Python SDK

The W&B Python SDK provides the `Sandbox` interface you use later to create and interact with the Serverless Sandbox. Install it using `pip`:

```bash 
pip install wandb
```

### Log in and authenticate with W&B

> The Serverless Sandbox runs under your W&B account, so you must be authenticated before you can create one. If you haven't done so already, log in to W&B. Use the `wandb login` CLI command and follow the prompts to log in to your W&B account:

Reviewer: ```suggestion
W&B Serverless Sandboxes run under your W&B account, so you must authenticate before you create one. Use the `wandb login` CLI command and follow the prompts to log in:
```

Let's call out "W&B" since folks might be aware of CW Sandboxes which may cause confusion.

Author: Applied in 1159ffaa8.

```bash
wandb login
```

See the [`wandb login`](/models/ref/cli/wandb-login) reference documentation for more information about how W&B searches for credentials.

## Copy the training script and dependencies

> In this step, you prepare the three files that the sandbox needs: a requirements file, a hyperparameters file, and the training script itself. Expand the following dropdown to access the code required for this tutorial. Copy and paste the code into three separate files in the same directory as this tutorial. In the next section, you run a script that reads in these files and trains a PyTorch model within a Serverless Sandbox environment.

Reviewer: ```suggestion
Prepare the three files required for this tutorial: a requirements file, a hyperparameters file, and a training script. Expand the following dropdown, then copy each code sample into a separate file in the same directory as this tutorial.

In the next section, you run a script that reads these files and trains a PyTorch model in a W&B Serverless Sandbox.
```

Author: Applied in 1159ffaa8.

## Create the sandbox and run the training script

> With your training files in place, you can now drive the sandbox from a single Python script. The following code snippet shows how to create a sandbox and copy the training script and dependencies into it. It then runs the training script and downloads the generated model file. The next section explains the code line by line.

Reviewer: ```suggestion
With your training files in place, create and manage a W&B Serverless Sandbox from a single Python script. The following code snippet creates a sandbox, copies the training script and dependencies into it, runs the training script, and downloads the generated model file.

The next section explains the code line by line.
```

"Drive"?

Author: Applied in 1159ffaa8; removed drive.
